06LessonSix/in-class-project/css/intel-style.css

```diff
@@ -8,43 +8,42 @@ header, main, footer {
     margin: auto;
 }
 
-!header {
+header {
     background-color: #0071c5;
     padding: 10px;
 }
 
-header-#h1 {
+header h1 {
     font-weight: 100;
     font-size: 60px;
     color: #fff;
 }
 
-body.nav-link. {
+.nav-link {
     color: #fff;
     font-size: 25px;
     margin-right: 15px;
 }
-main < h2 {
+h2 {
     font-size: 28px;
     font-weight: 200;
 }
 
-main aside h3 {
+aside h3 {
     font-size: 24px;
     font-weight: 200;
 }
 
-main > p {
+main < p {

```

Comment:
I don't think the symbol `<`  can be used here. I believe this selector just needed to have the `>` symbol removed. Look at the HTML and you will find no direct child `p` of `main`